tests/Behat/Context/Ui/RefundingContext.php

```diff
@@ -203,6 +214,14 @@ public function shouldNotBeAbleToRefundUnitWithProduct(int $unitNumber, string $
         Assert::false($this->orderRefundsPage->isUnitWithProductAvailableToRefund($productName, $unitNumber-1));
     }
 
+    /**
+     * @Then I should still be able to refund order shipment with :paymentMethodName payment
+     */
+    public function shouldStillBeAbleToRefundOrderShipment(): void
+    {
+        Assert::true($this->orderRefundsPage->isOrderShipmentAvailableToRefund());
```

Comment:
Since :paymentMethodName parameter is not used, maybe we can get rid of it?

Reply:
We cannot, as the step name is the same as in `@application` test, where this payment method is relevant :/